fe/fe-core/src/main/java/com/starrocks/external/hive/HiveMetastoreApiConverter.java

```diff
@@ -79,11 +87,17 @@ public static HiveTable toHiveTable(Table table, String catalogName) {
                 .setFullSchema(toFullSchemasForHiveTable(table))
                 .setTableLocation(table.getSd().getLocation())
                 .setCreateTime(table.getCreateTime());
+
+        Map<String, String> tableParams = table.getParameters() == null ? ImmutableMap.of() : table.getParameters();
+        if (isDeltaLakeTable(tableParams)) {
```

Comment:
abstract a function named 'toTableLocation()'. we may try our best to keep clear here.